packages/hoprd/src/index.ts

```diff
@@ -63,6 +63,15 @@ const argv = yargs
     describe: 'Updates the port for the healthcheck server',
     default: 8080
   })
+  .option('publicLogs', {
```

Comment:
What the PR is doing is classic log forwarding with ceramic as the log sink. Thus I would also adapt the naming to be closer to that. E.g. forwardLogs and forwardLogsEndpoint.

Reply:
Yep, I like that naming better indeed